### DISCUSSION TITLE
submitted proposal to the website assuming it could be marked as a draft

### DISCUSSION THREAD

**Discussion by chebro** [cncf/mentoring#626] [Q&A]
i submitted a proposal to the gsoc website because the contributor guidelines mentioned that we could upload the proposal and mark it as draft. however i did not find any such option after submitting it to the website. as of now i have withdrawn my proposal. what is the recommended way to get proposals reviewed by the mentors?

**Comment by nate-double-u** [comment]
Hi @chebro!

I think your question is similar to this one in the FAQ:
[Should I send proposals directly to the mentoring organizations?](https://developers.google.com/open-source/gsoc/faq#should_i_send_proposals_directly_to_the_mentoring_organizations)

Proposals need to be submitted through the program site -- as you have done. Reviews, however, are done the same way that you have previously communicated with the project mentors.
[
Proposals can be edited as many times as you like](https://google.github.io/gsocguides/student/writing-a-proposal#submit-a-proposal-early) before the deadline (April 19 - 18:00 UTC)

**Reply by chebro** [reply]
thank you! i have submitted my proposal through the website and modified the title to indicate that it is a draft.
